### ISSUE TITLE
[1.2.0] - PacBio Only

### ISSUE THREAD

**Issue by DLBPointon** [sanger-tol/treeval#297]
### Description of feature

There has been a request for a PacBio only run. This is something not currently supported, however, should be simple enough to implement. Related to #115

**Comment by DLBPointon** [comment]
Add line `if (params.subworkflows.contains('no_hic') {}` to line 246 to treeval.nf

**Comment by yumisims** [comment]
does this mean no juicer map?

**Comment by DLBPointon** [comment]
> does this mean no juicer map?

Yep, no juicer. The requester isn't and won't be generating any HiC at all.

**Comment by yumisims** [comment]
I think we should use no_juicer instead of no_hic ?

**Comment by DLBPointon** [comment]
We already have a `--juicer` flag for juicer specifically. `--no_hic` would turn off Pretext (+ associated pretext accessory files), HiGlass and Juicer.
